.circleci/check-cpp-format.sh

```diff
@@ -0,0 +1,175 @@
+stratum/hal/lib/phal/onlp/onlp_wrapper_mock.h
+stratum/hal/lib/phal/onlp/onlp_wrapper.h
+stratum/hal/lib/phal/optics_adapter_test.cc
+stratum/hal/lib/phal/phal_backend_interface.h
+stratum/hal/lib/phal/phal_cli.cc
+stratum/hal/lib/phal/phal_test.cc
+stratum/hal/lib/phal/phal.cc
+stratum/hal/lib/phal/phal.h
+stratum/hal/lib/phal/phaldb_service.h
+stratum/hal/lib/phal/sfp_adapter_test.cc
+stratum/hal/lib/phal/sfp_adapter.cc
+stratum/hal/lib/phal/sfp_adapter.h
+stratum/hal/lib/phal/tai/tai_interface_mock.h
+stratum/hal/lib/phal/tai/tai_interface.h
+stratum/hal/lib/phal/tai/tai_optics_datasource_test.cc
+stratum/hal/lib/phal/tai/tai_optics_datasource.h
+stratum/hal/lib/phal/tai/tai_phal_test.cc
+stratum/hal/lib/phal/tai/tai_phal.cc
+stratum/hal/lib/phal/tai/tai_phal.h
+stratum/hal/lib/phal/tai/tai_switch_configurator_test.cc
+stratum/hal/lib/phal/tai/tai_switch_configurator.cc
+stratum/hal/lib/phal/tai/tai_switch_configurator.h
+stratum/hal/lib/phal/tai/taish_client.cc
+stratum/hal/lib/phal/tai/taish_client.h
+stratum/hal/lib/phal/test_util.h
+stratum/hal/lib/pi/pi_node.cc
+stratum/hal/lib/pi/pi_node.h
+stratum/hal/stub/embedded/main.cc
+stratum/lib/constants.h
+stratum/lib/constants.h
+stratum/lib/libcproxy/libcwrapper.cc
+stratum/lib/libcproxy/libcwrapper.cc
+stratum/lib/libcproxy/libcwrapper.h
+stratum/lib/libcproxy/libcwrapper.h
+stratum/lib/libcproxy/passthrough_proxy.cc
+stratum/lib/libcproxy/passthrough_proxy.cc
+stratum/lib/libcproxy/passthrough_proxy.h
+stratum/lib/libcproxy/passthrough_proxy.h
+stratum/lib/test_utils/p4_proto_builders.cc
+stratum/lib/test_utils/p4_proto_builders.cc
+stratum/p4c_backends/common/backend_extension_interface.h
+stratum/p4c_backends/common/backend_pass_manager.h
+stratum/p4c_backends/common/midend_interface.h
+stratum/p4c_backends/common/p4c_front_mid_interface.h
+stratum/p4c_backends/fpm/bcm/bcm_target_info.cc
+stratum/p4c_backends/fpm/bcm/bcm_target_info.h
+stratum/p4c_backends/fpm/bcm/bcm_tunnel_optimizer.h
+stratum/p4c_backends/fpm/condition_inspector.h
+stratum/p4c_backends/fpm/hidden_static_mapper.h
+stratum/p4c_backends/fpm/internal_action.h
+stratum/p4c_backends/fpm/meta_key_mapper.cc
+stratum/p4c_backends/fpm/pipeline_optimizer.h
+stratum/p4c_backends/fpm/simple_hit_inspector.h
+stratum/p4c_backends/fpm/table_type_mapper.h
+stratum/p4c_backends/fpm/target_info.cc
+stratum/p4c_backends/fpm/target_info.h
+stratum/p4c_backends/fpm/tunnel_optimizer_interface.h
+stratum/p4c_backends/fpm/tunnel_type_mapper.h
+stratum/p4c_backends/test/test_p4c_main.cc
+stratum/p4c_backends/test/test_target_info.cc
+stratum/p4c_backends/test/test_target_info.h
+stratum/testing/tests/bcm_sim_test_fixture.cc
+stratum/testing/tests/bcm_sim_test_fixture.h
+stratum/testing/tests/bcm_sim_test.cc
+stratum/testing/tests/test_main.cc
+stratum/tools/gnmi/gnmi_cli.cc
+\0
+EOF
+
+echo "$CHANGED_FILES" | xargs -t -n1 clang-format --style=file -i
+echo "$KNOWN_FILES" | xargs -t -n1 clang-format --style=file -i
```

Comment:
```suggestion
echo "$KNOWN_FILES\n$CHANGED_FILES" | sort -u | xargs -t -n1 clang-format --style=file -i
```

We can avoid potentially duplicate work.